apps/src/studio/studio.js

```diff
@@ -1653,10 +1679,28 @@ var TICKS_BEFORE_FACE_SOUTH = 5;
  * Given direction/emotion/tickCount, calculate which frame number we should
  * display for sprite.
  */
```

Comment:
Would be useful to add a description of possible opts. See http://usejsdoc.org/tags-param.html section "parameters with properties" for a description of how to do this.